### PR TITLE
MODKBEKBJ-523 Implement /eholdings/uc endpoint

### PR DESCRIPTION
**Pull request by psmagin**
## Purpose
Implement `/eholdings/uc` endpoint

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -677,6 +682,11 @@
         "kb-ebsco.kb-credentials.access-types.item.delete"
       ]
     },
+    {
```
Comment (natali-zaitseva) [inline descriptors/ModuleDescriptor-template.json:685]: will this permission be included in some of the "all" permission set?

File: src/main/java/org/folio/rest/impl/EholdingsUsageConsolidationImpl.java [hunk 2]
```diff
@@ -23,39 +23,39 @@
...
 
   @Override
-  public void getEholdingsKbCredentialsUcById(String credentialsId, Map<String, String> okapiHeaders,
-                                              Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-    settingsService.fetchByCredentialsId(credentialsId, okapiHeaders)
+  public void getEholdingsUc(Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
+                             Context vertxContext) {
+    settingsService.fetchByUser(okapiHeaders)
       .thenAccept(ucSettings -> asyncResultHandler.handle(succeededFuture(
         GetEholdingsKbCredentialsUcByIdResponse.respond200WithApplicationVndApiJson(ucSettings))))
```
Comment (natali-zaitseva) [inline src/main/java/org/folio/rest/impl/EholdingsUsageConsolidationImpl.java:49]: Different response class. should be GetEholdingsUcResponse

File: descriptors/ModuleDescriptor-template.json (hunk 1 above)
Comment (psmagin) [reply descriptors/ModuleDescriptor-template.json:685]: Thanks! Fixed

File: src/main/java/org/folio/rest/impl/EholdingsUsageConsolidationImpl.java (hunk 2 above)
Comment (psmagin) [reply src/main/java/org/folio/rest/impl/EholdingsUsageConsolidationImpl.java:49]: Fixed

**Review by natali-zaitseva — APPROVED** [review]

**Pull request merged**